WORKSPACE

```diff
@@ -126,3 +126,8 @@ maven_install(
     ],
 )
 
+android_sdk_repository(
```

Comment:
My local tests seem to have required this and I'm not sure why the buildkite test runs are fine without it.

Reply:
Can you remove build_tools_version and api_level, please? The CI may not have these versions pinned.

Reply:
Done.

Reply:
Your local tests needed this? Which tests? This shouldn't be necessary. :/